### DISCUSSION TITLE
转化成web时，抽取react、react-dom有问题

### DISCUSSION THREAD

**Discussion by enmeen** [NervJS/taro#11963] [Q & A] [answered]
要使用taro打包后的web页，发现taro默认会将react、react-dom打入bundle。通过配置编译文件做了处理，并且手动在html中引入了react相关的cdn文件
` chain.merge({
        externals: {
          react: "React",
          'react-dom': 'ReactDOM',
        }
      })`
发现会有报错
<img width="630" alt="image" src="https://user-images.githubusercontent.com/17080071/174754641-3afe670c-632d-49a2-9462-7010efaa43e0.png">

**Comment by Chen-jj** [comment, marked as answer]
cdn 的 React 版本改用 17 试试，因为 Taro 3.5 开始才支持 React 18
